products/jbrowse-web/src/tests/TextSearching.test.tsx

```diff
@@ -52,9 +52,10 @@ test('test trix from lgv header', async () => {
   fireEvent.mouseDown(input)
   fireEvent.change(input, { target: { value: 'eden.1' } })
   fireEvent.keyDown(auto, { key: 'Enter', code: 'Enter' })
+  // now there are more options, so it will open a dialog instead
```

Comment:
might be good to keep this one intact. we can probably make a solution where we just do not index the plaintext version of the volvox.gff so that no dialog pops up

Reply:
Restored all the tests that were modified by indexing the plain text tracks.